Packs/BitDam/pack_metadata.json

```diff
@@ -2,10 +2,10 @@
     "name": "BitDam",
     "description": "BitDam secure email gateway protects from advanced content-borne threats with the most accurate prevention of known and unknown threats, at their source.",
     "support": "partner",
-    "currentVersion": "1.0.0",
+    "currentVersion": "1.0.1",
     "author": "BitDam",
-    "url": "",
-    "email": "info@bitdam.com",
+    "url": "https://www.bitdam.com",
```

Comment:
@mchasepan How are you with this?

Reply:
[Other issue](https://github.com/demisto/etc/issues/29340) should have changed that.... @ShahafBenYakir